src/bun.js/bindings/webcore/WebSocket.cpp

```diff
@@ -1051,158 +1253,158 @@ void WebSocket::didFailWithErrorCode(int32_t code)
     // invalid_response
     case 1: {
         static NeverDestroyed<String> message = MAKE_STATIC_STRING_IMPL("Invalid response");
-        didReceiveMessageError(1002, message);
+        didReceiveClose(false, 1002, message);
```

Comment:
I would feel a little better if this was an enum instead of a boolean because false and true are very ambiguous here.